src/components/Modals/AddClientModal.jsx

```diff
@@ -92,6 +94,7 @@ const AddClientModal = ({ showAddClientModal, setShowAddClientModal }) => {
 
     runNotification(
       `Adding "${userObject.givenName} ${userObject.familyName}" to client list...`,
+      'succes',
```

Comment:
may be best to save `success` for when it's actually successful (line 120)? and maybe these (when it's trying to do what the user asked) are better suited for either `warning` or `info` colors?

Reply:
actually another option to the `warning` or `info` colors would be taking off the `variant="filled` prop (see my comment below on `BasicNotification.jsx` - but haven't thought about how this would be implemented... some sort of ternary on this prop maybe.